### PR TITLE
release(v0.41.0): prepare release

### PR DESCRIPTION
**Pull request by Unix4ever**
This is the official v0.41.0 release.

### CODE REVIEW

**Review by frezbo — APPROVED** [review]
nothing major to report in release notes?

**Comment by Unix4ever** [comment]
yeah, no user facing features for this release. Mostly bugfixes.

File: LICENSE [hunk 1]
```diff
@@ -7,7 +7,7 @@ Licensed Work:        Omni
                       The Licensed Work is (c) 2024 Sidero Labs, Inc.
 Additional Use Grant: None
 
-Change Date:          2028-08-16
```
Comment (smira) [inline LICENSE:10]: we probably shouldn't change this on the release branch (not critical, but still)

release branch is a snapshot of `main`, so we should rather update `main`

**Pull request merged**